src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, color 300ms;
```

Comment:
Everything looks good maybe you can use variables for color and transition duration.

Reply:
Yes, I'll consider that point. Thanks